src/java/org/apache/cassandra/db/guardrails/GuardrailsConfigService.java

```diff
@@ -0,0 +1,216 @@
+    }
+
+    @Override
+    public int getColumnsPerTableAbortThreshold()
+    {
+        return (int) config.getColumnsPerTable().getAbortThreshold();
+    }
+
+    @Override
+    public void setColumnsPerTableAbortThreshold(int threshold)
+    {
+        config.getColumnsPerTable().setAbortThreshold(threshold);
+    }
+
+    @Override
+    public int getSecondaryIndexesPerTableWarnThreshold()
+    {
+        return (int) config.getSecondaryIndexesPerTable().getWarnThreshold();
+    }
+
+    @Override
+    public void setSecondaryIndexesPerTableWarnThreshold(int threshold)
+    {
+        config.getSecondaryIndexesPerTable().setWarnThreshold(threshold);
+    }
+
+    @Override
+    public int getSecondaryIndexesPerTableAbortThreshold()
+    {
+        return (int) config.getSecondaryIndexesPerTable().getAbortThreshold();
+    }
+
+    @Override
+    public void setSecondaryIndexesPerTableAbortThreshold(int threshold)
+    {
+        config.getSecondaryIndexesPerTable().setAbortThreshold(threshold);
+    }
+
+    @Override
+    public int getMaterializedViewsPerTableWarnThreshold()
+    {
+        return (int) config.getMaterializedViewsPerTable().getWarnThreshold();
+    }
+
+    @Override
+    public void setMaterializedViewsPerTableWarnThreshold(int threshold)
+    {
+        config.getMaterializedViewsPerTable().setWarnThreshold(threshold);
+    }
+
+    @Override
+    public int getMaterializedViewsPerTableAbortThreshold()
+    {
+        return (int) config.getMaterializedViewsPerTable().getAbortThreshold();
+    }
+
+    @Override
+    public void setMaterializedViewsPerTableAbortThreshold(int threshold)
+    {
+        config.getMaterializedViewsPerTable().setAbortThreshold(threshold);
+    }
+
+    @Override
+    public String getTablePropertiesDisallowed()
+    {
+        return toCSV(config.getTableProperties().getDisallowed());
+    }
+
+    @Override
+    public void setTablePropertiesDisallowed(String properties)
+    {
+        setTablePropertiesDisallowed(fromCSV(properties));
+    }
+
+    public void setTablePropertiesDisallowed(String... properties)
+    {
+        setTablePropertiesDisallowed(ImmutableSet.copyOf(properties));
+    }
+
+    public void setTablePropertiesDisallowed(Set<String> properties)
+    {
+        config.getTableProperties().setDisallowedValues(properties);
+    }
+
+    @Override
+    public String getTablePropertiesIgnored()
+    {
+        return toCSV(config.getTableProperties().getIgnored());
+    }
+
+    @Override
+    public void setTablePropertiesIgnored(String properties)
+    {
+        setTablePropertiesIgnored(fromCSV(properties));
+    }
+
+    public void setTablePropertiesIgnored(String... properties)
+    {
+        setTablePropertiesIgnored(ImmutableSet.copyOf(properties));
+    }
+
+    public void setTablePropertiesIgnored(Set<String> properties)
+    {
+        config.getTableProperties().setIgnoredValues(properties);
+    }
```

Comment:
you could expose via the mbean, many tools do support this.

Reply:
Same as before, I've exposed the one for `Set<String>` and renamed the CSV one.